src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/util/StorageStubHttpClient.java

```diff
@@ -14,28 +15,63 @@
     private static final String LIST_CONTAINERS = "?comp=list";
     private static final String LIST_ONE_BLOB = "/bulkscan?include=&restype=container&comp=list";
     private static final String LIST_ZERO_BLOBS = "/empty?include=&restype=container&comp=list";
+    private static final String FILE_1 = "/bulkscan/file1.zip";
+    private static final String FILE_3 = "/bulkscan/file3.zip";
+    private static final String FILE_CAUSING_404 = "/bulkscan/causes_404.zip";
+    private static final String FILE_CAUSING_500 = "/bulkscan/causes_500.zip";
```

Comment:
just a note - while creating and amending this file i have maintained the alphabetical order to those private static declarations. just a not.